### ISSUE TITLE
after fresh install ( - ova binary) - can't login cause of IPv6 Address

### ISSUE THREAD

**Issue by alzmich** [agnitas-org/openemm#6]
Hi,

after a fresh OVA Installation i wasn't able to login to the frontend with the generated emm-master password.
Digging into the tomcat logfiles i got:

Caused by: java.sql.SQLException: Data too long for column 'ip_address' at row 1
Query is: INSERT INTO login_track_tbl (ip_address, login_status, username) VALUES (?, ?, ?), parameters ['XXXX:XXXX:XXXX:XXXX:XXXX:XXXX:XXXX:XXXX',10,'emm-master']

... looking into the database i figured out, the ip_address column of login_track_tbl table is only a varchar(20), so i altered it to a varchar(64) - since we have a ipv6 only network.
 - after this it worked.

i looked into the emm-mysql-fulldb-basic.sql and found a few other ip_address cols that where varchar(20) and changed them too.

 - see patch below.

Regards,

Michael.

```
diff --git a/frontend/sql/mysql/emm-mysql-fulldb-basic.sql b/frontend/sql/mysql/emm-mysql-fulldb-basic.sql
index 7191641..996b975 100644
--- a/frontend/sql/mysql/emm-mysql-fulldb-basic.sql
+++ b/frontend/sql/mysql/emm-mysql-fulldb-basic.sql
@@ -914,7 +914,7 @@ CREATE UNIQUE INDEX jobqueue$description$unique ON job_queue_tbl (description);

 CREATE TABLE login_track_tbl (
        login_track_id             INT(11) NOT NULL AUTO_INCREMENT COMMENT 'unique ID, use login_track_tbl_seq',
-       ip_address                 VARCHAR(20) COMMENT '[secret_data] address where login-request came from',
+       ip_address                 VARCHAR(64) COMMENT '[secret_data] address where login-request came from',
        creation_date              TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'login-request timestamp',
        login_status               INT(11) COMMENT '10 = successfull, 20 = failed, 40 = successfull but while IP is locked',
        username                   VARCHAR(200) COMMENT '[secret_data] emm-user name used in request',
@@ -923,7 +923,7 @@ CREATE TABLE login_track_tbl (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT '[secret_data] any login-request, successfull or not, is stored here (for a certain time)';

 CREATE TABLE login_whitelist_tbl (
-       ip_address                 VARCHAR(20) NOT NULL COMMENT '[secret_data] IP to whitelist',
+       ip_address                 VARCHAR(64) NOT NULL COMMENT '[secret_data] IP to whitelist',
        description                VARCHAR(200) NOT NULL COMMENT 'reason to whitelist'
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT '[secret_data] listed Adresses will never get Login locked';

@@ -1290,7 +1290,7 @@ CREATE TABLE swyn_click_tbl (
        network_id                 VARCHAR(20) COMMENT 'This is the content of the field "SOURCE" of the SWYN_TBL to identify the network from where the click came.',
        mailing_id                 INT(11) COMMENT 'references mailing (mailing_tbl.mailing_id)',
        customer_id                INT(11) COMMENT 'references recipient who shared the newsletter (customer_xxx_tbl.customer_id)',
-       ip_address                 VARCHAR(15) COMMENT 'The ip-address of the clicker',
+       ip_address                 VARCHAR(64) COMMENT 'The ip-address of the clicker',
        timestamp                  TIMESTAMP NULL COMMENT 'The timestamp of the click',
        selector                   VARCHAR(2000) COMMENT 'If the used anon.view is restricted to a part of the newsletter, this contains the selector used to limit the output of the content'
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci COMMENT 'This table records all clicks for displaying a shared newsletter on a network.';
```

**Comment by agnitas-org** [comment]
Thanks for your feedback! We will fix that asap.

**Comment by agnitas-org** [comment]
Fixed with frontend version 19.10.000.131.
